File: LeetCode75/Array/1768-merge-alternately/1768-merge-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {

        string merged="";
        int max_len = max(word1.length(), word2.length());

        for ( int i = 0; i < max_len; i++)
        {
            if ( i < word1.length())
            {
                merged += word1[i];
            }
            if ( i < word2.length())
            {
                merged += word2[i];
            }
        }

        return merged;

    }
};

File: LeetCode75/Array/1768-merge-alternately/1768-merge-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        merged =""
        max_len = max(len(word1), len(word2))

        for i in range(max_len):
            if ( i < len(word1)):
                merged += word1[i]
            if (i < len(word2)):
                merged += word2[i]
        

        return merged
                




        
